sdk/communication/communication-identity/test/public/utils/recordedClient.ts

```diff
@@ -84,7 +84,7 @@ export function createRecordedCommunicationIdentityClientWithToken(
   const recorder = record(context, environmentSetup);
   let credential: TokenCredential;
   const endpoint = parseConnectionString(env.COMMUNICATION_CONNECTION_STRING).endpoint;
-  if (isPlaybackMode()) {
+  if (isPlaybackMode() && isNode) {
```

Comment:
This is because browser tests don't work for token credential right?

Reply:
yup!

Reply:
Isn't ClientSecretCredential working?

@sadasant

Reply:
Is this the pattern that's supposed to be done for browser tests? We could switch to using it

Reply:
Are you skipping the browser live tests with identity?

If so, you don't have to. 
`ClientSecretCredential` would work in the browser with disabled web security through karma.conf.

Reply:
@DominikMe I know when we were originally setting up the we made the decision to not use `ClientSecretCredential` because it would not be an accurate representation of what Contoso would be using / we want to encourage them to use `DefaultAzureCredential`.

Reply:
My point was to use the same code that our users would write in our tests. Can we use `DefaultAzureCredential` and disable web security so that the chained credential picks up `ClientSecretCredential` under the hood?

Reply:
@HarshaNalluru we're going to address the use of ClientSecretCredential in browser tests in a separate issue later. We're also considering defering that work until identity v2 is released and using the new solution for that